### PR TITLE
New adapter: GreedyGame

### PR DESCRIPTION
**Pull request by imedvedko**
Docs: https://github.com/prebid/prebid.github.io/pull/4850

### CODE REVIEW

File: static/bidder-params/greedygame.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "properties": {
+    "publisherId": {
+      "type": ["integer", "string"],
+      "description": "Publisher ID",
+      "minimum": 1,
+      "pattern": "^[1-9][0-9]*$"
+    }
+  },
+
+  "required": ["publisherId"]
```
Comment (gargcreation1992) [inline static/bidder-params/greedygame.json:16]: This adapter is an alias of `limelightDigital` adapter. Please use the exact same bidder params as `limelightDigital`.

File: static/bidder-info/greedygame.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+endpoint: "http://ads-pbs.rtb-greedygame.com/openrtb/{{.PublisherID}}"
+maintainer:
+  email: "engineering@project-limelight.com"
```
Comment (gargcreation1992) [inline static/bidder-info/greedygame.yaml:3]: Just a note for other reviewers - This email address has been validated on Aug 28th so skipping to send a verification email this time.

**Review by gargcreation1992 — COMMENTED** [review]
Requesting you to please use the same bidder params as limelightDigital.

File: static/bidder-params/greedygame.json (hunk 1 above)
Comment (imedvedko) [reply static/bidder-params/greedygame.json]: It is fixed in https://github.com/prebid/prebid-server/pull/3083/commits/3114ba42cb13d5c3a953ecf73cec40df31b42a66 commit

**Review by gargcreation1992 — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**